### PR TITLE
update ncs to `3203d1319278a4b8abd02e53043b6487ec8f7bc5`

### PR DESCRIPTION
**Pull request by maciejbaczmanski**
OpenThread version inside: `a759a4a09f9fe5bdf757c470aa86dffdc7a349f1`

### CODE REVIEW

File: script/make-firmware.bash [hunk 1]
```diff
@@ -354,8 +356,10 @@ build_ncs()
     local rcp_1_2=("/tmp/ncs_rcp_1_2" "samples/openthread/coprocessor/" "${script_dir}/../config/ncs/overlay-rcp-1_2.conf")
     local cli_1_3=("/tmp/ncs_cli_1_3" "samples/openthread/cli/" "${script_dir}/../config/ncs/overlay-cli-1_3.conf")
     local rcp_1_3=("/tmp/ncs_rcp_1_3" "samples/openthread/coprocessor/" "${script_dir}/../config/ncs/overlay-rcp-1_3.conf")
+    local cli_1_4=("/tmp/ncs_cli_1_3" "samples/openthread/cli/" "${script_dir}/../config/ncs/overlay-cli-1_4.conf")
```
Comment (sunytt) [inline script/make-firmware.bash:359]: should it be /tmp/ncs_cli_1_4 here?

File: script/make-firmware.bash [hunk 2]
```diff
@@ -354,8 +356,10 @@ build_ncs()
...
+    local rcp_1_4=("/tmp/ncs_rcp_1_3" "samples/openthread/coprocessor/" "${script_dir}/../config/ncs/overlay-rcp-1_4.conf")
```
Comment (sunytt) [inline script/make-firmware.bash:360]: should it be /tmp/ncs_cli_1_4 here?

File: script/make-firmware.bash [hunk 3]
```diff
@@ -341,6 +341,8 @@ package_ncs()
     distribute "/tmp/ncs_rcp_1_2/coprocessor/zephyr/zephyr.hex" "ot-rcp" "1.2" "${timestamp}" "${commit_id}"
     distribute "/tmp/ncs_cli_1_3/cli/zephyr/zephyr.hex" "ot-cli-ftd" "1.3" "${timestamp}" "${commit_id}"
     distribute "/tmp/ncs_rcp_1_3/coprocessor/zephyr/zephyr.hex" "ot-rcp" "1.3" "${timestamp}" "${commit_id}"
+    distribute "/tmp/ncs_cli_1_4/cli/zephyr/zephyr.hex" "ot-cli-ftd" "1.3" "${timestamp}" "${commit_id}"
```
Comment (sunytt) [inline script/make-firmware.bash:344]: should it be "1.4" here?

File: script/make-firmware.bash [hunk 4]
```diff
@@ -341,6 +341,8 @@ package_ncs()
...
+    distribute "/tmp/ncs_rcp_1_4/coprocessor/zephyr/zephyr.hex" "ot-rcp" "1.3" "${timestamp}" "${commit_id}"
```
Comment (sunytt) [inline script/make-firmware.bash:345]: should it be "1.4" here or?

File: script/make-firmware.bash [hunk 5]
```diff
@@ -341,6 +341,8 @@ package_ncs()
...
 }
```
Comment (sunytt) [inline script/make-firmware.bash:346]: It looks like package_ncs() is called 5 times with the type and version on line 369, but the it doesn't use the type and version from the caller and distributes all possible types and versions.

Should package_ncs() only be called once?

**Review by sunytt — COMMENTED** [review]
Could you update the CHANGELOG.txt to track this change?

File: script/make-firmware.bash (hunk 1 above)
Comment (maciejbaczmanski) [reply script/make-firmware.bash]: right! copied too fast :D

File: script/make-firmware.bash (hunk 5 above)
Comment (maciejbaczmanski) [reply script/make-firmware.bash:346]: yes, https://github.com/openthread/ot-reference-release/pull/86 resolves that, but needs to be merged after this PR and https://github.com/openthread/ot-reference-release/pull/85

**Review by sunytt — APPROVED** [review]
LGTM, thanks!

**Review by jwhui — APPROVED** [review]

**Pull request merged**